spec/helpers/games_helper_spec.rb

```diff
@@ -11,5 +11,5 @@
 #   end
 # end
 RSpec.describe GamesHelper, type: :helper do
```

Comment:
You might want to just delete these unused files if you don't have a clear use for them. They can always be re-added.